nixos/modules/system/boot/systemd.nix

```diff
@@ -869,11 +869,15 @@ in
       "sysctl.d/50-coredump.conf".source = "${systemd}/example/sysctl.d/50-coredump.conf";
       "sysctl.d/50-default.conf".source = "${systemd}/example/sysctl.d/50-default.conf";
 
+      "tmpfiles.d/home.conf".source = "${systemd}/example/tmpfiles.d/home.conf";
       "tmpfiles.d/journal-nocow.conf".source = "${systemd}/example/tmpfiles.d/journal-nocow.conf";
+      "tmpfiles.d/portables.conf".source = "${systemd}/example/tmpfiles.d/portables.conf";
```

Comment:
https://github.com/NixOS/systemd/blob/nixos-v234/tmpfiles.d/portables.conf

Reply:
This link 404s

Reply:
This file is only present in more recent versions: https://github.com/NixOS/systemd/tree/nixos.v243.4/tmpfiles.d

Reply:
Note that portabled doesn't even work on nixos currently. It crashes somewhere deep in systemd .